Options/Applicative.hs

```diff
@@ -14,6 +15,11 @@ module Options.Applicative (
   -- | The standard @Applicative@ module is re-exported here for convenience.
   module Control.Applicative,
 
+  -- | Re-export Data.Monoid's (<>) operator for convenience.
```

Comment:
This is a bit evil as then optparse-applicative's exposed API depends on its transitive dependencies configuration. I.e., if you happen to use `base-4.5` or later (even if you don't declare direct dependency on `base`) you get `(<>)` in scope, if use anything else, importing `Options.Applicative` does not provide `((<>))`. 

So this is nasty API-variability which causes us Hackage Trustees quite some headaches, as we'd need to add constraints that are way beyond the kind of things we can edit into `.cabal` files retroactively.

I'd strongly advise against this. If you want to export conditionally, better drop support for base < 4.5 and thereby avoid the conditional  export altogether :-)

(or you could provide a local `<>` definition for when `base` doesn't provide it; this runs the risk of potential clashes with other local `<>` definitions but it's the lesser evil IMO)